src/allmydata/util/_python3.py

```diff
@@ -0,0 +1,10 @@
+"""Track the port to Python 3."""
+
+# Keep these sorted alphabetically, to reduce merge conflicts:
+PORTED_MODULES = [
+    "allmydata.util.humanreadable",
+]
+
+PORTED_TEST_MODULES = [
+    "allmydata.test.test_humanreadable",
+]
```

Comment:
There is currently a `src/allmydata/ported-modules.txt` which is used as an input to `misc/python3/tahoe-depgraph.py`.  It doesn't differentiate between ported implementation and test modules though, nor does it have such affordances as comments...  Perhaps this Python module is better than a text file?  I'm not sure what future plans you might have for this.

Add this stuff to that file or change `tahoe-depgraph.py` to use this file and delete the other one - your choice.

Reply:
I will switch to `ported-modules.txt`.

Reply:
Or, hm.

Reply:
I guess I'll delete `ported-modules.txt`, having something more structured seems (potentially) useful.